public/app/plugins/datasource/prometheus/completer.ts

```diff
@@ -9,6 +9,24 @@ export class PromCompleter {
   }
 
   getCompletions(editor, session, pos, prefix, callback) {
+    let token = session.getTokenAt(pos.row, pos.column);
+
+    switch (token.type) {
+      case 'label.name':
+        callback(null, ['instance', 'job'].map(function (key) {
```

Comment:
would be cool to parse out the preceding metric name and lookup label values here :)

Reply:
Yeah, I want to do it.
I will try it in this weekend :)